init-tools.sh

```diff
@@ -13,8 +12,8 @@ __DOTNET_PATH=$__TOOLRUNTIME_DIR/dotnetcli
 __DOTNET_CMD=$__DOTNET_PATH/dotnet
 if [ -z "$__BUILDTOOLS_SOURCE" ]; then __BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json; fi
 export __BUILDTOOLS_USE_CSPROJ=true
-__BUILD_TOOLS_PACKAGE_VERSION=$(cat $__scriptpath/BuildToolsVersion.txt)
-__DOTNET_TOOLS_VERSION=$(cat $__scriptpath/DotnetCLIVersion.txt)
+__BUILD_TOOLS_PACKAGE_VERSION=$(cat $__scriptpath/BuildToolsVersion.txt | tr -d '\r\n')
```

Comment:
Are you able to build Windows enlistment on Unix with this change?

IIRC, folks have tried to do similar changes in the past, but it was hard to make it work completely.

Reply:
IIRC, I needed this for `build.sh` to work for me. Otherwise a trailing newline caused the script to fail. I don't really know what you mean by building the windows enlistment because I don't imagine it calling a bash script. I was able to run `./build.sh` and then publish a sample corert app with my local build. It was all done inside a docker container (I created a separate PR for that change).

Reply:
This change is compensating for Windows new lines on Unix.

One typically ends up with Windows new lines on Unix by:
1. Clone on Windows where git is configured to use Windows new lines by default
2. Map the directory into a Unix docker container, and build inside the docker container
Have you done something like this?

Reply:
I did both. But I don't think the volume mount on docker would change the files.